### ISSUE TITLE
Task: Help Center - pass the plan name, not slug when submitting an email ticket

### ISSUE THREAD

**Issue by alshakero** [Automattic/wp-calypso#68294]
### Details

See point 2 here: p7DVsv-fpN-p2#comment-42223

The wording should be exactly like the FAB because they use it to auto-tag tickets. You can get the wording from here: https://github.com/Automattic/wp-calypso/blob/trunk/client/me/help/help-contact/index.jsx#L177-L180

**Comment by kriskarkoski** [comment]
We rely on this text for tagging plan length so it should be in the following format we get from FAB:

`Plan: ID - Name in English (Plan length)`

For example:

`Plan: 1013 - Premium (Monthly subscription)`

**Comment by klimeryk** [comment]
Ooof, really? TIL. How does that work? That sounds quite brittle and not sure if accurate. The plan that is being mentioned here is the one that the current site has. But the plan tag that we add in Zendesk (and send when creating the ticket on the backend) is based on the top paid plan on the whole account. So it might not always be the same plans (and lengths).

**Comment by kriskarkoski** [comment]
@klimeryk Yep, it's not ideal. Basically when we launched monthly and two-year plans we didn't have the resources to implement this properly (see pcbrnV-wF-p2) so we just added the plan name and it's ID to the body of the message, then I setup routing on the Zendesk side to tag tickets, which I refreshed a few months ago in p8wKgj-2ME-p2

I think for Help Center launch we should just keep that as-is, but we should probably have a separate conversation around what we base support on to make sure we're all aligned and how we store the highest plan versus the plan on the site they select. What I'm uncertain of is I don't know if anyone has ever looked at plan term data so do we even need it? I assumed at the time we'd analyze for differences in behavior in support load by plan term, but that hasn't happened to my knowledge.

If that conversation sounds good as a next step I can kick that off within a few weeks on our p2.
